src/Type.php

```diff
@@ -135,6 +148,14 @@ private static function keyAndValueTypeFromArray(array $value): array
 
     public function __toString(): string
     {
+        if ($this->name === 'Struct') {
```

Comment:
Warum verwendest da nicht `$this->isStruct()`?

Reply:
Because that's an extra stack frame that can be avoided. `isStruct()` is only meant for external use. It's obviously just a micro-optimization, but performance is particularly important to us here.

Reply:
👍